src/main/java/xyz/nucleoid/bedwars/game/active/shop/BwItemShop.java

```diff
@@ -12,24 +12,209 @@
+                    addUpgrade(shop, upgrades, UpgradeType.SHEARS, new LiteralText("Add Shears"));
+                }
+            });
+        }
+        else if(pageIndex == 5){
+            return ShopUi.create(new LiteralText("Archery"), shop -> {
+                BwParticipant participant = game.getParticipant(player);
+                if(participant != null){
+                    // Navigation
+                    addNavbar(shop, player, game);
+                    // Bows and arrows
+                    shop.addItem(new ItemStack(Items.BOW), Cost.ofGold(12));
+                    shop.addItem(ItemStackBuilder.of(Items.BOW).addEnchantment(Enchantments.POWER, 2).build(), Cost.ofGold(24));
+                    shop.addItem(ItemStackBuilder.of(Items.BOW).addEnchantment(Enchantments.PUNCH, 1).build(), Cost.ofEmeralds(6));
+                    shop.addItem(new ItemStack(Items.ARROW, 8), Cost.ofGold(2));
+                }
+            });
+        }
+        else if(pageIndex == 6){
+            return ShopUi.create(new LiteralText("Utilities and Potions"), shop -> {
+                BwParticipant participant = game.getParticipant(player);
+                if(participant != null){
+                    // Navigation
+                    addNavbar(shop, player, game);
+                    // Potions
+                    shop.addItem(PotionUtil.setPotion(new ItemStack(Items.POTION), Potions.STRONG_LEAPING), Cost.ofEmeralds(1));
+                    shop.addItem(PotionUtil.setPotion(new ItemStack(Items.POTION), Potions.STRONG_SWIFTNESS), Cost.ofEmeralds(1));
+                    shop.addItem(PotionUtil.setPotion(new ItemStack(Items.POTION), Potions.INVISIBILITY), Cost.ofEmeralds(2));
+                    shop.addItem(new ItemStack(Blocks.TNT), Cost.ofGold(8));
+                    shop.addItem(new ItemStack(Items.FIRE_CHARGE).setCustomName(new LiteralText("Fireball")), Cost.ofIron(40));
+                    shop.addItem(new ItemStack(Items.ENDER_PEARL), Cost.ofEmeralds(4));
+                    shop.addItem(new ItemStack(Items.WATER_BUCKET), Cost.ofGold(10));
+                    shop.addItem(new ItemStack(Items.LAVA_BUCKET), Cost.ofGold(24));
+                    shop.addItem(new ItemStack(Items.GOLDEN_APPLE), Cost.ofGold(3));
+                    shop.addItem(new ItemStack(BwItems.CHORUS_FRUIT).setCustomName(new LiteralText("Chorus Fruit")), Cost.ofGold(8));
+                    shop.addItem(new ItemStack(BwItems.BRIDGE_EGG), Cost.ofEmeralds(2));
+                    shop.addItem(new ItemStack(BwItems.MOVING_CLOUD), Cost.ofEmeralds(2));
+                }
+            });
+        }
+        return old(player, game);
+    }
+
+    private static void addNavbar(ShopBuilder shop, ServerPlayerEntity player, BwActive game) {
+        //Creates a page navigation bar at the top of the shop
+        shop.add(ShopEntry.ofIcon(Blocks.END_STONE).withName(new LiteralText("Blocks")).withCost(Cost.free()).onBuy(page -> {
```

Comment:
Is it possible to indicate in the navigation bar which tab is currently selected with an enchantment glint? Not sure if enchantment glints can be applied to anything- alternatively it could maybe switch out the item.

Reply:
Could probably just make it have prot 1 and stop the enchant from showing on the tooltip?

Reply:
I'll add that soon